region/lowernorfair/east.json

```diff
@@ -1656,11 +1656,11 @@
                     "heatProof",
                     "canHeatRun"
                   ]},
-                  "Super",
                   {"cost": [
-                    {"heatFrames": 1200}
+                    {"heatFrames": 400}
```

Comment:
Finally, there should probably a way to use this by killing the enemies as well? We could have it require heatproof at worst.

Reply:
That's the whole issue with duplicating the reqs for a simple runway. Where to draw the line or should the full thing be at both places?

Reply:
If duplication becomes an issue, we could consider using a helper.

Reply:
Do you want to move this concern to another issue? The PR is already bloated enough as is

Reply:
(Issue #125 created)